tests/unit/PartsKitTest.php

```diff
@@ -79,4 +79,11 @@ public function testGetImage()
 
         $this->assertInstanceOf('craft\elements\Asset', $image);
     }
+
+    public function testGetFirstNavUrl()
+    {
+        Craft::$app->request->setUrl('/parts-kit');
+
+        $this->assertEquals('/parts-kit/button/blue', Module::$instance->partsKit->getFirstNavUrl());
+    }
```

Comment:
I tried to do a functional test, but codeception was like "nahhh gonna throw a random twig error" (because of the `redirect` tag